gmusicapi/protocol/mobileclient.py

```diff
@@ -185,7 +189,6 @@
         'albums': {'type': 'array', 'items': sj_album, 'required': False},
         'topTracks': {'type': 'array', 'items': sj_track, 'required': False},
         'total_albums': {'type': 'integer', 'required': False},
-        'artistBio': {'type': 'string', 'required': False},
```

Comment:
It was in the dictionary twice (my editor just flagged it). Look on line 183/187.